hypervisor/scripts/makefile/config.mk

```diff
@@ -146,7 +146,7 @@ endif
 ifeq ($(BOARD), apl-nuc)
   override BOARD := nuc6cayh
 else ifeq ($(BOARD), kbl-nuc-i7)
-  override BOARD := nuc7i7dnb
+  override BOARD := nuc11tnbi5
```

Comment:
Let's remove [this entire block](https://github.com/projectacrn/acrn-hypervisor/blob/1316b398b84342214e89b77956ddf219dd457981/hypervisor/scripts/makefile/config.mk#L145-L150), we do not support these legacy names and their associated boards anyway (APL and KBL)

Reply:
Updated.